code/game/objects/items/storage/wallets.dm

```diff
@@ -37,9 +37,7 @@
 		/obj/item/reagent_containers/dropper,
 		/obj/item/reagent_containers/syringe,
 		/obj/item/reagent_containers/pill,
-		/obj/item/screwdriver,
-		/obj/item/stamp),
-		list(/obj/item/screwdriver/power))
```

Comment:
correct me if I'm wrong, but isn't this line here specifically to forbid power drills from being stored in wallets?

Reply:
I missed that.  I'm so used to seeing `blacklist` when referring to restricted objects in the code.